Packages/Newtonsoft.Json-for-Unity.Converters/Configuration/KeyedConfig.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Newtonsoft.Json.UnityConverters.Configuration
+{
+#pragma warning disable CA2235 // Mark all non-serializable fields
+    [Serializable]
+    public struct KeyedConfig : IEquatable<KeyedConfig>
+    {
+        public string key { get; set; }
+
+        public ConfigType type { get; set; }
+
+        public bool boolean { get; set; }
+
+        public int integer { get; set; }
+
+        public float number { get; set; }
+
+        public string text { get; set; }
+
+        public override bool Equals(object obj)
+        {
+            return obj is KeyedConfig config && Equals(config);
+        }
+
+        public bool Equals(KeyedConfig other)
+        {
+            return key == other.key &&
+                   type == other.type &&
+                   boolean == other.boolean &&
+                   integer == other.integer &&
+                   number == other.number &&
```

Comment:
Codacy found an issue: [Do not check floating point equality with exact values, use a range instead.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5865055)